### PR TITLE
Update build-a-native-executable.md

### PR DESCRIPTION
**Pull request by TharmiganK**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md [hunk 1]
```diff
@@ -46,6 +46,16 @@ The only downside is that the GraalVM native image build is a highly complicated
 
 From Ballerina 2201.3.0 (SwanLake) onwards, Ballerina supports GraalVM AOT compilation to generate standalone executables by passing the native flag in the build command: `bal build --native`. The generated executable contains the modules in the current package, their dependencies, Ballerina runtime, and statically linked native code from the JDK.
 
+> **Info:** Apart from Ballerina runtime and standard libraries the following Ballerina external libraries are GraalVM compatible :
```
Comment (praneesha) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:49]:
```suggestion
> **Info:** Apart from the Ballerina runtime and standard libraries, the following Ballerina external libraries are GraalVM compatible :
```

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md [hunk 2]
```diff
@@ -46,6 +46,16 @@ The only downside is that the GraalVM native image build is a highly complicated
...
+>  - [NATS](https://central.ballerina.io/ballerinax/nats)
+>  - [Kafka](https://central.ballerina.io/ballerinax/kafka)
+>  - [RabbitMQ](https://central.ballerina.io/ballerinax/rabbitmq)
+>  - [JDBC](https://central.ballerina.io/ballerinax/java.jdbc)
+>  - [MSSQL](https://central.ballerina.io/ballerinax/mssql)
```
Comment (praneesha) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:54]:
```suggestion
>  - [`mssql`](https://central.ballerina.io/ballerinax/mssql)
```

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md [hunk 3]
```diff
@@ -46,6 +46,16 @@ The only downside is that the GraalVM native image build is a highly complicated
...
+>  - [MySQL](https://central.ballerina.io/ballerinax/mysql)
+>  - [OracleDB](https://central.ballerina.io/ballerinax/oracledb)
```
Comment (praneesha) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:56]:
```suggestion
>  - [`oracledb`](https://central.ballerina.io/ballerinax/oracledb)
```

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md [hunk 4]
```diff
@@ -60,12 +70,12 @@ To complete this part of the guide, you need:
 ### Configure GraalVM
 
 1. Install GraalVM if you have not done it already.
-    - Install GraalVM with one line
+    - Install GraalVM on Linux, macOS, and Windows(via Git Bash, Cygwin, or WSL) with one line
```
Comment (praneesha) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:73]:
```suggestion
    - Install GraalVM on Linux, macOS, and Windows (via Git Bash, Cygwin, or WSL) with one line
```
What do we mean by `one line` here? Better if we can explain it more.

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md [hunk 5]
```diff
@@ -60,12 +70,12 @@ To complete this part of the guide, you need:
...
       ```
       $ bash <(curl -sL https://get.graalvm.org/jdk)  graalvm-ce-java11-22.3.0
       ```
       > **Tip:** The above command installs the native-image tool, which is required to generate the native images along with GraalVM.
-    - For additional information, see [Get Started with GraalVM](https://www.graalvm.org/22.2/docs/getting-started/)
+    - For additional information, see [Get Started with GraalVM](https://www.graalvm.org/22.3/docs/getting-started)
```
Comment (praneesha) [inline swan-lake/learn-the-platform/native-support/build-a-native-executable.md:78]:
```suggestion
    - For additional information, see [Get Started with GraalVM](https://www.graalvm.org/22.3/docs/getting-started).
```

File: swan-lake/learn-the-platform/native-support/build-a-native-executable.md (hunk 4 above)
Comment (TharmiganK) [reply swan-lake/learn-the-platform/native-support/build-a-native-executable.md:73]: The command below is a single command to install GraalVM and native-image tool. I think we don't need to emphasize it here. We can just say : 
```suggestion
    - Execute the following command to install GraalVM on Linux, macOS, and Windows(via Git Bash, Cygwin, or WSL)
```
@shafreenAnfar @praneesha WDYT?

Comment (praneesha) [reply swan-lake/learn-the-platform/native-support/build-a-native-executable.md]: Let's just mention the command. No need to say `one line`.

Comment (TharmiganK) [reply swan-lake/learn-the-platform/native-support/build-a-native-executable.md]: @praneesha Fixed via [92ebef7](https://github.com/ballerina-platform/ballerina-dev-website/pull/5905/commits/92ebef771a0c2052d07c49cd872ac69bcccd230b)

**Review by praneesha — APPROVED** [review]

**Pull request merged**